hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/client/HddsClientUtils.java

```diff
@@ -71,7 +71,7 @@ private static void doNameChecks(String resName, String resType) {
     if (resName.length() < OzoneConsts.OZONE_MIN_BUCKET_NAME_LENGTH ||
         resName.length() > OzoneConsts.OZONE_MAX_BUCKET_NAME_LENGTH) {
       throw new IllegalArgumentException(resType +
-          " length is illegal, " + "valid length is 3-63 characters");
+          " name '" + resName + "' is of illegal length, " + "valid length is 3-63 characters");
     }
```

Comment:
- I think we could further improve it by checking and rejecting too short and too long values separately.
- I suggest changing "illegal" to "invalid".
- Should we cap too long invalid value in the message at some reasonable length?

Reply:
- Indeed! That would be better.
- Yes, I think so too. If the special character check made sense, then this does as well.

Thank you, added both.